deploy/start.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+
+export ASSISTED_SERVICE_URL="${ASSISTED_SERVICE_URL:-http://localhost:8090}"
```

Comment:
If the ASSISTED_SERVICE_URL env-var isn't persisted in the Dockerfile. Would we be able to override it?